apps/site/lib/site_web/templates/partial/_hidden_icons.html.eex

```diff
@@ -22,7 +22,7 @@
     end
   %>
   <%=
-    for mode <- [:subway, :commuter_rail, :bus, :ferry] do
+    for mode <- [:subway, :commuter_rail, :bus, :ferry, :silver_line] do
       content_tag(:div, [mode_icon(mode, :small)], id: ["icon-feature-", Atom.to_string(mode)])
```

Comment:
I found the culprit that makes `mode_icon(:silver_line, :small)` always return the bus icon!! 

https://github.com/mbta/dotcom/blob/867be60394ddf736e264eac875d9eb6aeb959ea5/apps/site/lib/site_web/views/helpers.ex#L79-L81